src/models/cart-model.js

```diff
@@ -211,7 +211,9 @@ const CartModel = BaseModel.extend({
     const newLength = newLineItems.length;
 
     if (newLength < oldLength) {
-      this.attrs.line_items = newLineItems;
+      this.attrs.line_items = newLineItems.map(item => {
```

Comment:
single-line fat arrows without the braces implicitly return. I like the "as-needed" option for this rule but I'll check the shopify eslint config http://eslint.org/docs/rules/arrow-body-style